app/routes/photos/components/GalleryPictureEditModal.tsx

```diff
@@ -80,20 +78,8 @@ const GalleryPictureEditModal = ({
             id="gallery-picture-description"
           />
           <Field
-            label={
-              <Flex>
-                <div>Synlig for offenligheten</div>
-                <div
-                  style={{
-                    marginLeft: '5px',
-                  }}
-                >
-                  <Tooltip content="Om bildet skal være synlig for brukere som ikke har tilgang til å redigere albumet.">
-                    <Icon name="information-circle-outline" size={20} />
-                  </Tooltip>
-                </div>
-              </Flex>
-            }
+            label="Synlig for offenligheten"
+            description="Om bildet skal være synlig for brukere som ikke har tilgang til å redigere albumet"
             placeholder="Synlig for alle brukere"
```

Comment:
The placeholder here does not actually do anything since it's a checkbox. Removed it now.

Do you still think the label and description are saying different things? I suppose it depends on how you interpret "offentligheten". I suggest rewriting the label to "Synlig for alle brukere" and then remove the need for a description. Thoughts?
```suggestion
            label="Synlig for alle brukere"
```

Reply:
Sounds good! Alternatively it could be possible to add a description of "Hvis ikke er den kun synlig for brukere med redigeringstilgang" if that doesn't get too long.. (as the alternative would be left unexplained if not) but either way I'd say it's an improvement

Reply:
Nvm or just keep the description as it is - didn't see the current description while I was writing my previous comment

Reply:
I feel like the description would be smør på flesk. The field is already in an editor, so it is obviously available to people with editorial rights.

Reply:
Fair nuff